src/components/Badge/Badge.css.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable sort-keys-fix/sort-keys-fix */
+import { recipe } from "@vanilla-extract/recipes";
+import { vars } from "../../utils/theme.css";
+
+export const BadgeContainer = recipe({
+  base: {
+    alignItems: "center",
+    borderRadius: 20,
+    display: "inline-flex",
+    fontFamily: "Inter, -apple-system, BlinkMacSystemFont, sans-serif",
+    fontSize: 12,
+    fontWeight: 400,
+    gap: 8,
+    justifyContent: "center",
+    lineHeight: 1.5,
+    padding: "8px 16px",
+  },
+  variants: {
+    theme: {
+      dark: {
+        backgroundColor: vars.colors.deepgreen[100],
```

Comment:
it is ok to use directly the color name here, and not having a token?

Reply:
we were checking it with @Serse2 , on Figma most of the stuff uses the primitive tokens rather than the domain ones, as overall "agreement" of the library we should take the info from the Figma as "truth" and eventually push the designers to update it if something isn't right.

TL;DR we shouldn't "guess" which token is the correct one